target/linux/ath79/image/nand.mk

```diff
@@ -1,13 +1,13 @@
 # attention: only zlib compression is allowed for the boot fs
 define Build/zyxel-buildkerneljffs
-	rm -rf  $(KDIR_TMP)/zyxelnbg6716
-	mkdir -p $(KDIR_TMP)/zyxelnbg6716/image/boot
-	cp $@ $(KDIR_TMP)/zyxelnbg6716/image/boot/vmlinux.lzma.uImage
+	rm -rf  $@.tmp
+	mkdir -p $@.tmp/image/boot
+	cp $@ $@.tmp/image/boot/vmlinux.lzma.uImage
```

Comment:
I understand that we need the boot subdir, but do we need the image subdir as well or could we just use $@.tmp/boot instead?

Reply:
Thanks for the hint, you are right. It's a leftover from old image generation tools.